pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -138,7 +138,7 @@ func (c *ClusterController) cleanUpJobContainer(cluster *cephv1.CephCluster, mon
 		SecurityContext: securityContext,
 		VolumeMounts:    volumeMounts,
 		Env:             envVars,
-		Args:            []string{"ceph", "clean"},
+		Args:            []string{"ceph", "clean", "disks"},
```

Comment:
In addition to disks, this command also cleans the host path, right? How about this command name?
```suggestion
		Args:            []string{"ceph", "clean", "host"},
```